agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -264,13 +262,18 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
         .addPropertiesCustomizer(new AiConfigCustomizer())
         .addLogRecordProcessorCustomizer(
```

Comment:
Suggestion: an extract method could be applied to this customizer, and  `addLogRecordProcessorCustomizer` could be used next to `.addLogRecordExporterCustomizer`. In this way it would be easier to notice the whole picture for logging.